src/components/create.vue

```diff
@@ -243,55 +243,54 @@ export default {
         this.$refs.errorModal.show();
         return;
       }
+
+      let blockchain = getBlockchain(this.selectedChain);
+
       try {
-        apkey = PrivateKey.fromWif(this.activepk)
-          .toPublicKey()
-          .toString(this.selectedChain);
-        mpkey = PrivateKey.fromWif(this.memopk)
-          .toPublicKey()
-          .toString(this.selectedChain);
+        apkey = blockchain.getPublicKey(this.activepk);
+        mpkey = blockchain.getPublicKey(this.memopk);
```

Comment:
This part will do for now but we should probably make number of keys, key "names" and what they can be used for, part of the chain configuration.  Adding a steem-based chain and not getting a "posting" key option is bad user experience

Reply:
That is very true!

Reply:
I suggest to keep active / memo for now, built that feature out in new PR